postcss.config.js

```diff
@@ -0,0 +1,5 @@
+/** @format */
+
+module.exports = {
+	plugins: [ require( 'autoprefixer' ) ],
+};
```

Comment:
Since this is just a couple lines, what do you think about moving this to `webpack.config.js` for clarity?

```js
{
  loader: 'postcss-loader',
  options: {
    plugins: [ require( 'autoprefixer' ) ],
  }
}
```

Reply:
Good point, done in https://github.com/Automattic/wp-calypso/pull/26724/commits/3d743fded9f35fa781effc8f4fcb524479db3c80